app/src/consumer/parser_impl_con.c

```diff
@@ -563,7 +563,7 @@ __Z_INLINE parser_error_t _readBody(parser_tx_t *v, CborValue *rootItem) {
             CHECK_CBOR_ERR(cbor_value_advance(&contents))
 
             break;
-        }
+        }/*
         case registryDeregisterEntity: {
```

Comment:
This seems like obsolete code. 
Can we remove it?

Reply:
I am waiting @tjanez to confirm that the ui flow is correct. I may be wrong and we still need it.

Reply:
I'm confirming the workflow is correct and matches https://github.com/Zondax/ledger-oasis/issues/135#issuecomment-893493809.

Reply:
Great!